corn/styles.black.css

```diff
@@ -0,0 +1,55 @@
+:root{
```

Comment:
The whole idea of having this separate CSS file was to override custom properties. You must have only rules for `:root` and nothing else, so when you uncomment `<link rel="stylesheet" href="styles.black.css" />` in HTML both CSS files will be applied. Try to fix it :)